example/src/App.js

```diff
@@ -52,6 +52,27 @@ const LogoImage = styled("img")`
   margin-right: 1em;
 `;
 
+function getBackEndUrl(urlString) {
+  // If given something falsey, we need to also return something falsey.
+  // Then ensures that we don't accidentally for the application to always be
+  // in editor/REST-mode.
```

Comment:
May be a grammar issue/typo here? @kylelaker

Reply:
```suggestion
  // This ensures that we don't accidentally force the application
  // to always be in editor/REST mode by creating a URL.
```

Oh wow what made me write that horror in the first place